docs/branches.md

```diff
@@ -0,0 +1,31 @@
+## Release branches
+
+There are two release branches in Toltec, [stable](https://toltec.delab.re/stable) and [testing](https://toltec.delab.re/testing).
+As a user, **you always want to use the stable branch**, which is the default one when following the install instructions.
+The testing branch exists solely for the repository maintainers to make sure that packages work correctly before distributing them to users.
+It may contain packages that could cause breakage if you install them on your device.
+
+### Adding or updating a package
+
+New packages or package updates are exclusively proposed through pull requests based on the testing branch.
+A proposal can be merged into testing after review from a maintainer and if it builds successfully in the CI.
+If it is a proposal for a new package, the maintainer who reviews the pull request becomes the maintainer for that package.
+If it updates an existing package, the maintainer of that specific package should do the review.
+
+### Moving a package from testing to stable
+
+Each Monday, a pull request containing selected updates from the testing branch and targeting the stable branch can be opened.
+This pull request can only contain package changes dating from the previous Friday or older.
+Each of these package changes must be tested by a maintainer different from the maintainer of the affected package.
+
+Here are important things to check when testing a package:
+
+1. The package should work as intended by upstream.
+2. It must not destroy any user data (e.g. from previous versions of the same package, from other packages, from the home directory).
+3. It must not break other packages.
```

Comment:
Some additional testing criteria that I suggest, following what we learned in the previous merge window.

```suggestion
1. It should work as intended by upstream.
2. It must not have any known major bugs.
3. It must not destroy any user data (e.g. from previous versions of the same package, from other packages, from the home directory).
4. It must not break other packages.
5. It must not lock the device in a state where it can not be used without troubleshooting from SSH.
```

Reply:
> 5. It must not lock the device in a state where it can not be used without troubleshooting from SSH.

just as a side note, i thought of another workaround last week for the issue with the device going into bad state: the device can usually be rebooted, so if he device gets into a bad state while away from your computer, you can still get out of the bad state by rebooting.

Reply:
That’s right. So maybe `5. It must not lock the device in a state where it can not be used without rebooting or troubleshooting from SSH.`

Reply:
> 2. It must not have any known major bugs.

I'd like it if we addressed how fixes can be brought over to stable. Based on the last merge, if a package of mine gets punted to the next merge, I'm unhappy compared to if a fix can be brought in during the testing window. I'm not sure if others feel this way